package.json

```diff
@@ -52,78 +52,78 @@
                 },
                 {
                     "id": "bookmarksHelpAndFeedback",
-                    "name": "Help and Feedback",
+                    "name": "%bookmarks.views.HelpAndFeedback.name%",
                     "visibility": "collapsed"
                 }
             ]
         },
         "viewsWelcome": [
             {
                 "view": "bookmarksExplorer",
-                "contents": "In order to use Bookmarks, you have to open a folder or workspace first.\n[Open a Folder](command:_bookmarks.openFolderWelcome)\n[Open a Workspace](command:workbench.action.openWorkspace)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+                "contents": "%bookmarks.viewsWelcome.bookmarksExplorer.contents.1%",
                 "when": "workbenchState == empty"
             },
             {
                 "view": "bookmarksExplorer",
-                "contents": "No bookmarks yet.\nIn order to use Bookmarks, you have to open a file in the editor.\n[Open a File](command:workbench.action.quickOpen)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+                "contents": "%bookmarks.viewsWelcome.bookmarksExplorer.contents.2%",
                 "when": "workbenchState != empty && !editorIsOpen"
             },
             {
                 "view": "bookmarksExplorer",
-                "contents": "No bookmarks yet.\nIn order to use Bookmarks, place the cursor in any location in the file and run the command:\n[Bookmarks: Toggle](command:bookmarks.toggle)\nTo learn more about how to use Bookmarks in VS Code [read the docs](http://github.com/alefragnani/vscode-bookmarks/#bookmarks).",
+                "contents": "%bookmarks.viewsWelcome.bookmarksExplorer.contents.3%",
```

Comment:
Avoid using `n` on ids. Use something meaningful, maybe based on the `when` clause, like `contents.whenHasFileOpen`